test/platform.ts

```diff
@@ -151,68 +161,102 @@ export class IOSEmulatorManager implements IEmulatorManager {
                 }
             })
             .then<string>(applicationLabel => {
-                return tu.TestUtil.getProcessOutput("xcrun simctl spawn booted launchctl stop " + applicationLabel, undefined, true);
-            })
-            .then<void>(output => {
-                console.log(output);
+                // kill the app if we found the process
+                return tu.TestUtil.getProcessOutput("xcrun simctl spawn booted launchctl stop " + applicationLabel, undefined);
+            }, (error) => {
+                // we couldn't find the app's process so it must not be running
+                return Q.resolve(error);
```

Comment:
well, this method is used not just to kill the app but also to guarantee an app isn't running. when the app is already gone, then the result state--the app is not running--is the same, so it's not necessary to reject because the desired end state of the promise has been maintained.